pom.xml

```diff
@@ -102,10 +102,10 @@ under the License.
   </properties>
 
   <scm>
-    <url>https://git-wip-us.apache.org/repos/asf?p=crunch.git</url>
-    <connection>scm:git:https://git-wip-us.apache.org/repos/asf/crunch.git</connection>
-    <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/crunch.git</developerConnection>
-    <tag>apache-crunch-0.13.0</tag>
+    <url>https://github.com/scalingdata/crunch</url>
+    <connection>scm:git:git@github.com:scalingdata/crunch.git</connection>
+    <developerConnection>scm:git:git@github.com:scalingdata/crunch.git</developerConnection>
+    <tag>0.13.0-hdp2.3.0-rocana</tag>
```

Comment:
This seems different from the naming convention we used in Impala. Our Impala branch is named `cdh5.4.5-rocana1.3.0`
Should I change the name we used in Impala.